src/data/structs/h1/tags/header.yml

```diff
@@ -0,0 +1,27 @@
+entry_type: header
```

Comment:
This file isn't currently used anywhere. I couldn't quite figure out how to get this to render the same way other tag parts do. It looks like that might take some actual JS work to get showing up properly.

Either way, I included it here for posterity.